lib/useClickedOutside.ts

```diff
@@ -0,0 +1,22 @@
+import { useEffect, RefObject } from 'react';
+
+const useClickededOutside = (ref: RefObject<any>, handler: () => void) => {
```

Comment:
Typo "clicked". Fix other occurances
```suggestion
const useClickedOutside = (ref: RefObject<any>, handler: () => void) => {
```